processor/metricstarttimeprocessor/internal/subtractinitial/adjuster.go

```diff
@@ -182,12 +184,14 @@ func adjustMetricExponentialHistogram(referenceValueTsm *datapointstorage.Timese
 
 			refTsi.ExponentialHistogram.RefCount, refTsi.ExponentialHistogram.RefSum, refTsi.ExponentialHistogram.RefZeroCount, refTsi.ExponentialHistogram.Scale = 0, 0, 0, currentDist.Scale()
 			refTsi.ExponentialHistogram.StartTime = resetStartTimeStamp
-			refTsi.ExponentialHistogram.RefPositive.BucketCounts().FromRaw(make([]uint64, currentDist.Positive().BucketCounts().Len()))
-			refTsi.ExponentialHistogram.RefNegative.BucketCounts().FromRaw(make([]uint64, currentDist.Negative().BucketCounts().Len()))
-			refTsi.ExponentialHistogram.PreviousPositive, refTsi.ExponentialHistogram.PreviousNegative = currentDist.Positive(), currentDist.Negative()
+			refTsi.ExponentialHistogram.RefPositive.BucketCounts = make([]uint64, currentDist.Positive().BucketCounts().Len())
```

Comment:
Do we also have to set `refTsi.ExponentialHistogram.RefPositive.Offset` and `refTsi.ExponentialHistogram.RefNegative.Offset` to 0 too?

(so that it doesn't get subtracted incorrectly on subsequent points?